### ISSUE TITLE
[ERROR]::MAIN: Unable to migrate config

### ISSUE THREAD

**Issue by lexguy2007** [clinton-hall/nzbToMedia#907]
When using to nzbtomedia to post process my TV shows from Nzbget I continue to get the following error: nzbToMedia: [18:18:41] [ERROR]::MAIN: Unable to migrate config file /Users/localpath/Library/Application Support/NZBGet/scripts/nzbToMedia-master/autoProcessMedia.cfg, exiting ... I cannot figure how what is causing it. Any ideas?

**Comment by clinton-hall** [comment]
this file `/Users/localpath/Library/Application Support/NZBGet/scripts/nzbToMedia-master/autoProcessMedia.cfg` has some bad formatting or an invalid config.

try removing, copying the .cfg.spec back to .cfg and then edit this file carefully to set up the required config.

If this doesn't work, email me the .cfg (XXXX out any api/pass) to `fock_wulf@hotmail.com` and I'll see if I can find the issue.

**Comment by lexguy2007** [comment]
I figured out what the issue was. I had been using Sonarr but didn't like it as much as SickRage. When I went to change and update the settings in nzbtomedia from within NZBGet, I blanked out the category under the Sonarr section. Everytime the the cfg was created, it had a " " as a category in the Sonarr section and it was causing the failure. I removed the nzbtomedia script and re-installed it and now it runs flawlessly. Thanks for your time in replying!!
